glib2/test/test_file_utils.rb

```diff
@@ -30,6 +30,10 @@ def test_gb
       assert_equal("1.0 GB", GLib.format_size(1000 * 1000 * 1000))
     end
 
+    def test_5gb
```

Comment:
It's better that test name means "guint32 over value".
Because "5" isn't important. "guint32 over value" is important.